doc/cloud-init.md

```diff
@@ -19,7 +19,7 @@ Possible actions include, for example:
 * Adding users
 * Enabling services
 * Running commands or scripts
-* Automatically growing the file system of a VM to the size of the disk
+* Automatically growing the file system of a VM to the size (quota) of the disk
```

Comment:
Is there any reason to prefer `size (quota)` over `size/quota`? Aesthetically I think I prefer the latter as parenthesizing it makes "quota" seem to be the less apt descriptor. My feeling is that we really do mean "quota" when we say "size".

Reply:
I went with that because all the configuration options, CLI commands, etc, use `size`, so it seems to deserve a heavier weight. Otherwise, I do understand what you mean about the word quota being a more accurate description.

Reply:
quota is a less apt description in some senses.

LXD models the concept of "volumes" that have a size. 

Depending on the storage driver the volume's size is enforced using the storage driver's quota functionality.

But LXD itself doesn't have storage quota support as a primary concept.

Instead the aim here is to allow users who are looking for "quota" to discover the "size" concept.